modules/components/pages/inputs/pg_stream.adoc

```diff
@@ -0,0 +1,459 @@
+= pg_stream
+// tag::single-source[]
+:type: input
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+Streams data changes from a PostgreSQL database using logical replication. There is also a configuration option to <<stream_snapshot,stream all existing data>> from the database.
+
+ifndef::env-cloud[]
+Introduced in version 4.40.0.
+endif::[]
+
+
+```yml
+# Configuration fields, showing default values
+input:
+  label: ""
+  pg_stream:
+    dsn: postgres://foouser:foopass@localhost:5432/foodb?sslmode=disable # No default (required)
+    include_transaction_markers: true
+    stream_snapshot: false
+    snapshot_memory_safety_factor: 1
+    snapshot_batch_size: 0
+    schema: public # No default (required)
+    tables: [] # No default (required)
+    checkpoint_limit: 1024
+    temporary_slot: false
+    slot_name: "" # No default (optional)
+    pg_standby_timeout: 10s
+    pg_wal_monitor_interval: 3s
+    max_parallel_snapshot_tables: 1
+    auto_replay_nacks: true
+    batching:
+      count: 0
+      byte_size: 0
+      period: "" # No default (optional)
+      check: "" # No default (optional)
+      processors: [] # No default (optional)
+```
+
+The `pg_stream` input uses logical replication to capture changes made to a PostgreSQL database in real time and streams them to Redpanda Connect. Redpanda Connect uses this replication method to allow you to choose which database tables in your source database to receive changes from. There are also <<choose-a-replication-mode,two replication modes>> to choose from, and an <<receive-toast-and-deleted-values,option to receive TOAST and deleted values>> in your data updates.
+
+== Prerequisites
+
+- PostgreSQL version 10 or later
+- Logical replication enabled on your PostgreSQL cluster
++
+To check whether logical replication is already enabled, run the following query:
++
+[,SQL]
+----
+SHOW wal_level;
+----
++
+If the `wal_level` value is `logical`, you can start to use this connector. Otherwise, choose from the following sets of instructions to update your replication settings.
+
+=== Self-Hosted PostgreSQL
```

Comment:
Since it's not a primary task, these could be tabs to avoid users having to scroll too much if they don't need these instructions.

Reply:
Agreed to add tabs enablement tasks so users only see what is relevant to them.